openpype/tools/pyblish_pype/model.py

```diff
@@ -616,9 +611,19 @@ def type(self):
 
     def data(self, role=QtCore.Qt.DisplayRole):
         if role == QtCore.Qt.DisplayRole:
+            label = None
             if settings.UseLabel:
-                return self.instance.data["label"]
-            return self.instance.data["name"]
+                label = (
+                    self.instance.data.get("label")
+                    or getattr(self.instance, "label", None)
```

Comment:
Has `instance.label` ever been I thing? I noticed that logic is also on the PluginModel and plugins do have `label` attribute but have never seen it on Pyblish instances.

Reply:
I'm actually not sure? I maybe added it in past (like 3-4 years ago) and was scared to remove later. If it's not a thing I can remove it.